susemanager/bin/pg-migrate-x-to-y.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+NV=""
+OV=""
+FAST_UPGRADE=""
+
+while getopts "s:d:f" o;do
+    case "${o}" in
+        s)
+            OV=${OPTARG}
+            ;;
+        d)  
+            NV=${OPTARG}
+            ;;
+       f)
+           FAST_UPGRADE=" --link"
+           ;;   
+       *)
+           echo "Usage: $0 [-s initial postgresql to migrate from] [-d destination postgresql to migrate to] [-f fast upgrade]"
+           echo "For example:"
+           echo "  $0 -s10 -d13"
```

Comment:
The other scripts call it with a space between option and parameter.
I am not a bash getopts guru, but does it also work when you call `$0 -s 10 -d 13` ?
I would think it is better to document it this way.
If this does not work, you call this script wrong from the others.

Reply:
It works with or without space. I just adjusted line 21 to match line 19 for consistency.